### PR TITLE
Update `CONTRIBUTING.md` and add `CODE_OF_CONDUCT.md`

### LINKED ISSUE #404: Adapt standard github configurations and contribution documentation
The ODK allows us to create a repo with standardized layouts for github issue templates, intstructions for contributors etc
https://github.com/INCATools/ontology-development-kit/

E.g.

https://github.com/INCATools/ontology-development-kit/blob/master/template/CONTRIBUTING.md.jinja2

(note this needs something about PRs)

We shoudl adapt for RO

### PR DESCRIPTION
**Pull request by anitacaron**
Fixes #404

### CODE REVIEW

**Review by matentzn — APPROVED** [review]
Fantastic @anitacaron looks great!

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,16 +1,89 @@
-# How to contribute to this project
+# Table Of Contents
 
-Please use our tracker for requests for new relations, modifications, clarifications, etc.
+- [Table Of Contents](#table-of-contents)
```
Comment (matentzn) [inline CONTRIBUTING.md:3]: There is not really a point the refer to the TOC from the TOC. Remove and reindent the rest?

Comment (anitacaron) [reply CONTRIBUTING.md:3]: I think it adds automatically because of the title

Comment (matentzn) [reply CONTRIBUTING.md:3]: who is "it" in "it automatically"?

**Review by jhpoelen — APPROVED** [review]

**Comment by jhpoelen** [comment]
@anitacaron to talk to @matentzn on closing out this issue.

**Pull request merged**